content/how-to/dedicated-hosting/delete-vpn-user-myrackspace-portal/index.md

```diff
@@ -38,24 +45,28 @@ This article describes how to delete a VPN user by using a ticket template.
    b. If you need to delete the VPN user from all firewalls on the account,
    ensure that you select all firewalls in this field.
 
+   <img width="410" alt="selectfirewall" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn6.png">
```

Comment:
This image has production device IDs, names and numbers showing. We probably want to manipulate the browser in the inspector and change the info before taking screenshot. I can show how to do that if needed.

Reply:
Image is changed as suggested.